src/utils/network/networking.ts

```diff
@@ -29,6 +29,11 @@ export async function httpPost(url: string, options?: AxiosRequestConfig, data?:
   return await axios.post(url, data, options);
 }
 
+export async function httpPatch(url: string, data: any, options?: AxiosRequestConfig): Promise<any> {
+  const response = await axios.patch(url, data, options);
+  return response.data ? response.data : null;
```

Comment:
Nitpick: could this be `response.data ?? null` ? If so i prefer that 😁

Reply:
I'll do that in the next PR. :muscle: